aws/resource_aws_default_vpc.go

```diff
@@ -50,44 +50,46 @@ func resourceAwsDefaultVpcCreate(d *schema.ResourceData, meta interface{}) error
 
 	resp, err := conn.DescribeVpcs(req)
 	if err != nil {
+		d.SetId("vpc-removed")
```

Comment:
i don't have a full grasp of where we are in the code, but do we want to return the err here, or maybe log it?